### DISCUSSION TITLE
폭탄 돌리기에서 처음에 폭탄을 받을 사람을 랜덤 지정하고 싶습니다.

### DISCUSSION THREAD

**Discussion by PH62mem** [naverz/zepeto-studio-kor#980] [5. Q&A]
폭탄 돌리기에 5명이 참가한다고 했을때 그 중에 한명이 처음 폭탄을 가지고 시작하게 하고 싶은데
구현할 방법을 모르겠습니다.

5명 중 한 명을 랜덤으로 고르고 폭탄 값을 줘야 하는데 어떻게 해야할까요?
도저히 감이 안잡힙니다....

참고 할 만한 코드나 방법이 있으면 써주시면 감사드리겠습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 멀티플레이를 통해 방에 있는 플레이어들의 목록을 받아오는 것까지는 구현 하셨나요?
랜덤에 대해서 도움을 드리자면 타입스크립트에서는 Math.random() 이 0 - 1 사이의 숫자가 생성됩니다.
따라서 0- 4 까지의 인덱스 사이 중 하나를 랜덤하게 pick하기 위해서는 
아래와 같은 방법으로 구현할 수 있습니다.

// min, max값 사이의 정수값을 반환하는 함수 선언
private Rand(min, max) {
      return Math.floor(Math.random() * (max - min)) + min;
}

//이후 아래와 같이 응용해서 구현하실 수 있습니다.
var seletionIndex = this.roomPlayersArray[this.rand(0,4)];

**Reply by PH62mem** [reply]
참가한 플레이어들의 목록을 받아오는건 했습니다만
그 중에 한 명을 랜덤하게 선택한 후에 다시 서버에 폭탄을 가진 사람을 어떻게 알려 줄 수 있나요?

**Reply by Zepeto-TechEvangelist** [reply]
room.Send()를 사용해서 서버쪽에 메세지를 보내면 됩니다. 
아래 멀티플레이 가이드 및 월드 로직 강의 영상이 도움이 될 것 같습니다.
https://studio.zepeto.me/guides/multiplay-room-message
